README.md

```diff
@@ -1,3 +1,6 @@
 # ajax-project
 
 A dynamic HTML, CSS, and JavaScript solo project.
+
+- Promise MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise
+- trim MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim
```

Comment:
I would move these to a different file `NOTES.MD` or something. `README` is where you can explain your project and how it works.